Corona-Warn-App/src/main/java/de/rki/coronawarnapp/statistics/ui/homecards/cards/KeySubmissionsCard.kt

```diff
@@ -22,7 +24,18 @@ class KeySubmissionsCard(parent: ViewGroup) :
     override val onBindData: HomeStatisticsCardsKeysubmissionsLayoutBinding.(
         item: StatisticsCardItem,
         payloads: List<Any>
-    ) -> Unit = { item, payloads ->
-        item.stats as KeySubmissionsStats
+    ) -> Unit = { item, _ ->
+        with(item.stats as KeySubmissionsStats) {
+            viewBinding.value.keySubmissionsLabel = getPrimaryLabel(context)
```

Comment:
Since this is an extension function on `HomeStatisticsCardsKeysubmissionsLayoutBinding` we can call `keySubmissionLabel` directly.